src/components/Header/header.module.css

```diff
@@ -0,0 +1,94 @@
+.header {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0 32px;
+    height: 64px;
+    font-family: 'Inter', sans-serif;
+    position: sticky;
+    top: 0;
+    background: #fff;
```

Comment:
i'm not sure how it should be, but it's looks like «transparent» background. Can you check it with designer?

<img width="1681" alt="Screen Shot 2021-12-09 at 18 31 01" src="https://user-images.githubusercontent.com/242514/145426275-f46f7620-61a8-4d63-9698-bd307a7fe0dc.png">

Reply:
Fixed, added `whiteHeader` prop to layout